### PR TITLE
Nc news 4

### PR DESCRIPTION
**Pull request by royr5**

### CODE REVIEW

File: models/articles.models.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 const db = require("../db/connection");
 
-exports.selectArticles = (id)=>{
-    return db.query(`SELECT * FROM articles WHERE article_id = $1`,[id])
-}
\ No newline at end of file
+exports.selectArticles = (id) => {
+  if (id < 1 || id > 13) {
+    return Promise.reject({ status: 404, msg: "article does not exist" });
+  }
```
Comment (Keb84) [inline models/articles.models.js]: This isn't very dynamic. What if you wanted to post an article and an id of 14 was created? How could you check the database to see if the article id exists instead?

File: __tests__/index.test.js [hunk 2]
```diff
@@ -37,19 +38,34 @@ describe("/api/topics", () => {
...
+      });
+  });
+  test("GET:404 sends an appropriate status and error message when given a valid but non-existent article id", () => {
+    return request(app)
+      .get("/api/articles/900")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("article does not exist");
+      });
+  });
```
Comment (Keb84) [inline __tests__/index.test.js:88]: Another test for status 400 is needed here 👀 what if the id was not a number eg; /api/articles/banana??

**Review by Keb84 — CHANGES_REQUESTED** [review]
Nice work so far. Please see my comments, make the changes and resubmit this ticket 🙂

**Review by nickkagame — APPROVED** [review]
All good Roshan!

**Pull request merged**